boskos/gcp/test-configs.yaml

```diff
@@ -0,0 +1,23 @@
+configs:
+- name: type2
+  needs:
+    type1: 1
+  config:
+    type: GCPResourceConfig
+    content: |
+      projectconfigs:
+      - type: type1
+        clusters:
+        - machinetype: n1-standard-2
+          numnodes: 4
+          version: 1.7
+          zone: us-central-1f
+        vms:
+        - machinetype: n1-standard-4
+          sourceimage: projects/debian-cloud/global/images/debian-9-stretch-v20180105
+          zone: us-central-1f
+          tags:
+          - http-server
+          - https-server
+          scopes:
+          - https://www.googleapis.com/auth/cloud-platform
```

Comment:
pro tip: config your editor to always add new line at the end

Reply:
Done